### ISSUE TITLE
Weird Line Heights issue with Capital Letters on certain font sizes

### ISSUE THREAD

**Issue by Lunchb0ne** [microsoft/terminal#11017]
### Windows Terminal version (or Windows build number)

1.10.1933.0

### Steps to reproduce

I'm using [JetBrainsMono NF](https://github.com/ryanoasis/nerd-fonts/blob/master/patched-fonts/JetBrainsMono/font-info.md) as my terminal font. On some sizes like 11pt the character heights are all inconsistent.

### Expected Behavior

Equal line Heights

### Actual Behavior

![image](https://user-images.githubusercontent.com/22198661/130487131-6dcf6ea5-9350-44a0-8b85-de8941ddb05e.png)

**Comment by zadjii-msft** [comment]
Curious, we've hit this before (#4367), but that was resolved over a year ago. Did this used to work in a previous version of the Terminal, with this font?

**Comment by Lunchb0ne** [comment]
> Curious, we've hit this before (#4367), but that was resolved over a year ago. Did this used to work in a previous version of the Terminal, with this font?

I'm not sure, I recently switched to this and it's broken on both the stable and preview Windows Terminal

**Comment by zadjii-msft** [comment]
I don't think this is the same as #11032 - that looks more like Cascadia Code was missing entirely. This is more just `JetBrains Mono NL Regular Nerd Font Complete.ttf` doesn't render exactly correct

**Comment by ttrautrim** [comment]
I tested it with the same font as described above. The issue only persisted in Windows Terminal using Powershell 7.2.2.0. There were no issues with the font in Windows Terminal using CMD or the default Windows Powershell. After upgrading Powershell to 7.3.0.3 (preview build) the issue stopped occurring.

Powershell 7.2.2.0
![powershell 7-2-2](https://user-images.githubusercontent.com/29348972/164763385-92dd8b65-faef-4dc6-bb06-b24f573f8234.png)

CMD
![cmd](https://user-images.githubusercontent.com/29348972/164763415-62b89d36-22a1-4392-ac04-e03f0cb98d60.png)

Default Windows Powershell
![windows_powershell](https://user-images.githubusercontent.com/29348972/164763466-35e4e51f-c679-46eb-98cc-9920cfa1825a.png)

Powershell 7.3.0.3
![powershell 7-3-0-3](https://user-images.githubusercontent.com/29348972/164765492-1f7ef1ad-e394-4dc4-876a-a189d238dc12.png)

**Comment by serd2011** [comment]
Is this still relevant? I cant seem to repro this on Preview 1.15.2282.0

**Comment by Lunchb0ne** [comment]
Sorry, I don't have access to Windows device for some time. I'll try to repro this ASAP.

**Comment by zadjii-msft** [comment]
Hey, if you've got a Windows machine around, can you double check if this repros still/? Terminal Preview 1.16 has a new rendering engine that might have fixed this.
